commands/output.go

```diff
@@ -244,7 +244,7 @@ func (d *droplet) Cols() []string {
 
 func (d *droplet) ColMap() map[string]string {
 	return map[string]string{
-		"ID": "ID", "Name": "Name", "PublicIPv4": "Public IPv4", "PublicIPv6": "Public IPv6",
+		"ID": "ID", "Name": "Name", "PublicIPv4": "Public IPv4", "PrivateIPv4": "Private_IPv4", "PublicIPv6": "Public IPv6",
 		"Memory": "Memory", "VCPUs": "VCPUs", "Disk": "Disk",
```

Comment:
Maybe you can just change `Private_IPv4` to `Private IPv4` so it looks better (e.g. like `Public IPv4`). This looks good but @braynl is someone who needs to review it :D